lua-libraries/.editorconfig

```diff
@@ -0,0 +1 @@
+root = true
```

Comment:
Is there not another way to configure ignoring a directory as part of the run? Having run this locally and looked at the output a bit I think there are going to be quite a few files we want this linter to ignore entirely.

Reply:
You can pass a regex to the `editorconfig-checker` command.

```text
  -exclude string
    	a regex which files should be excluded from checking - needs to be a valid regular expression
```

But to exclude a whole directory I think this is better because it also applies to other tools.

Reply:
Hmm, I can see the use in that.